CondaPkg.toml

```diff
@@ -0,0 +1,3 @@
+[deps]
+mrcfile = ""
+numpy = ""
```

Comment:
Personally I'd prefer these have lower and upper bounds, so that our test suite doesn't suddenly fail because of changes to either package's API. We can always manually update these from time to time if necessary.